datachecks/core/inspect.py

```diff
@@ -271,10 +271,10 @@ def _save_all_metrics(self, metric_values: List[MetricValue]):
         """
         self.metric_repository.save_all_metrics(metric_values)
         mm = self.metric_repository.get_all_metrics()
-        for k, v in mm.items():
-            print(k, v[0][0].json)
-            print(k, v[1][0].json)
-            print("====")
+        # for k, v in mm.items():
```

Comment:
@driptanil you can remove this comments, since It's been resolved in one other PR, you might face some conflict issue